openstack/cdn/v1/domains/results.go

```diff
@@ -59,6 +59,45 @@ type CdnDomain struct {
 	ModifyTime time.Time `json:"-"`
 }
 
+type DomainSourcesDetail struct {
+	OriginType          string `json:"origin_type"`
+	OriginAddr          string `json:"origin_addr"`
+	Priority            int    `json:"priority"`
+	ObsWebHostingStatus string `json:"obs_web_hosting_status"`
+	HttpPort            int    `json:"http_port"`
+	HttpsPort           int    `json:"https_port"`
+	HostName            string `json:"host_name"`
+	ObsBucketType       string `json:"obs_bucket_type"`
+}
+
+// CdnDomainDetail represents a CDN domain by domain name
+type CdnDomainDetail struct {
+	// the acceleration domain name ID
+	ID string `json:"id"`
+	// the acceleration domain name
+	DomainName string `json:"domain_name"`
+	// the service type, valid values are web, download, video and wholeSite
+	BusinessType string `json:"business_type"`
+	// the status of the acceleration domain name.
+	DomainStatus string `json:"domain_status"`
+	// the CNAME of the acceleration domain name
+	CName string `json:"cname"`
+	// the domain name or the IP address of the origin server
```

Comment:
Please verify the description information

Reply:
OK